### PR TITLE
Chapter5

### PR DESCRIPTION
**Pull request by Ruslan357**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (auraz) [inline .gitignore]: add newline

File: 5.14.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+        # Exercise 2
+#2.1
+def check_fermat(a,b,c,n):
+    if n > 2 and  a**n + b**n == c**n:
+        print("Holy smokes, Fermat was wrong!\n")
+    else:
+        print("No, that doesn’t work.\n")
```
Comment (auraz) [inline 5.14.py:22]: Looks good

File: 5.14.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+conversion()
+
+        # Exercise 3
+
+#3.1
+def is_triangle(segment_1, segment_2, segment_3):
+    if segment_1>segment_2+segment_3 or segment_2>segment_1+segment_3 or segment_3>segment_1+segment_2:
+        print("No\n")
+    else:
+        print("Yes\n")
```
Comment (auraz) [inline 5.14.py:40]: Looks good

**Review by auraz — CHANGES_REQUESTED** [review]
Minor comments, continue

File: .gitignore (hunk 1 above)
Comment (Ruslan357) [reply .gitignore]: Done!

File: 5.14.py (hunk 2 above)
Comment (Ruslan357) [reply 5.14.py:22]: Thank you

File: 5.14.py (hunk 3 above)
Comment (Ruslan357) [reply 5.14.py:40]: Thank you

**Pull request merged**